distribution/pom.xml

```diff
@@ -44,6 +49,7 @@
         <module>install-profiles</module>
         <module>ddf</module>
         <module>test</module>
+        <module>docs</module>
```

Comment:
@ricklarsen sorry I missed this earlier, but is there a reason you moved the docs module to be last? I believe the ddf module has a dependency on some of the documentation artifacts.